packages/trader/src/sass/app/modules/mt5/mt5-dashboard.scss

```diff
@@ -525,21 +529,17 @@
             &-data {
                 .mt5-account-card--paragraph {
                     @include typeface(--paragraph-left-bold-black, none);
-                    color: var(--text-prominent);
+                    color: var(--text-general);
                     line-height: 1.45;
                 }
```

Comment:
Isn't this removed inside `mt5-account-card.jsx`? I think we can get rid of it here.

Reply:
We still need this since some `p` tags are using it. We can remove it after changing all p tags to the `Text` component.